### ISSUE TITLE
RTX 3060 Memory Temp

### ISSUE THREAD

**Issue by Cyprez13** [trexminer/T-Rex#629]
RTX 3060 not showing memory temperature in Windows. Works fine on HWINFO64 and NiceHash.

**Comment by trexminer** [comment]
I don't think 3060s support memory junction temperature. Could you show a screenshot of HWInfo64 where you see it?

**Comment by Cyprez13** [comment]
![image](https://user-images.githubusercontent.com/89241372/134427799-860bc7ed-05ac-47d6-8ab3-295ba5038065.png)

![image](https://user-images.githubusercontent.com/89241372/134427847-6183c29d-9efe-4e79-9cbd-8cb477a3da84.png)

**Comment by trexminer** [comment]
That's what I expected. You're looking at the hot spot temperature, but the miner only displays memory junction temp
![image](https://user-images.githubusercontent.com/44735836/134440227-57ad2efd-c9b2-463e-a5ec-95b8412eb856.png)

**Comment by Cyprez13** [comment]
Fair enough, thank you. Just wanted to check because in my 2060, it's showing memory temp in T-Rex and HWINFO64 is showing Hot Spot Temp as well.

![image](https://user-images.githubusercontent.com/89241372/134440813-9fbfe175-976a-42a4-9a3c-ff3241c837de.png)

![image](https://user-images.githubusercontent.com/89241372/134440844-846d0335-5e62-4689-acaa-d72f37e21bd5.png)

**Comment by trexminer** [comment]
Ok, thanks, I didn't know we display something that's not actually there, haha. Will need to check what's going on.

**Comment by Cyprez13** [comment]
No worries, cheers.
